### PR TITLE
166667923-story(buyer signup): buyer signup

### PR DESCRIPTION
**Pull request by Codester291**
#### What does this PR do?
Registers Buyer
#### Description of Task to be completed?
Buyers should be able to sign up
#### How should this be manually tested?
clone or download
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166667923](https://www.pivotaltracker.com/story/show/166667923)
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/pages/buyer-signup/buyer-signup.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      <ion-input type="text" class="buyer" Placeholder="type your name"></ion-input>
+    </ion-col>  
+  </ion-row>
+  <ion-row>
+    <ion-col >
+      <ion-label class="buyerlbl">Email Address</ion-label>
+      <ion-input type="text" class="buyer" Placeholder="type your name"></ion-input>
+    </ion-col>  
+  </ion-row>
+  <ion-row>
```
Comment (JuwonAbiola) [inline src/pages/buyer-signup/buyer-signup.html]: please add stacked to the labels

File: src/pages/buyer-signup/buyer-signup.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    // }
+    .container{
+        position: relative;
+        left: 20%;
+        top: -10%;
+        right: 6.38%;
+        bottom: -8.33%;
+        width: 40px;
+        height: 40px; 
+        border-radius: 50%;
```
Comment (JuwonAbiola) [inline src/pages/buyer-signup/buyer-signup.scss]: please minimize your use of css

File: src/pages/buyer-signup/buyer-signup.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        border-radius: 5px;
+    }
+    .buyerlbl{
+        font-style: normal;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 19px;
+        color: #F2994A;
+    }
+    .someText{
```
Comment (JuwonAbiola) [inline src/pages/buyer-signup/buyer-signup.scss]: please use better naming convention next time

File: src/pages/buyer-signup/buyer-signup.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  Ionic pages and navigation.
+-->
+<ion-header style="background-color: #F44336;">
+
+  <ion-navbar>
+ 
+          <ion-icon ios="ios-arrow-round-back" md="md-arrow-round-back" slot="start" style="color: #fff"></ion-icon>
+        <div class="container">
+          <!-- <ion-title>buyer-signup-page</ion-title> -->
+          <img src="../../assets/imgs/sourcing produce logosource.svg">
```
Comment (JuwonAbiola) [inline src/pages/buyer-signup/buyer-signup.html]: please next time this can be reduced to  assets/logo/logo.svg

**Review by JuwonAbiola — CHANGES_REQUESTED** [review]

File: src/pages/buyer-signup/buyer-signup.html (hunk 1 above)
Comment (Codester291) [reply src/pages/buyer-signup/buyer-signup.html]: done!

File: src/pages/buyer-signup/buyer-signup.scss (hunk 2 above)
Comment (Codester291) [reply src/pages/buyer-signup/buyer-signup.scss]: what do you propose?

File: src/pages/buyer-signup/buyer-signup.scss (hunk 3 above)
Comment (Codester291) [reply src/pages/buyer-signup/buyer-signup.scss]: noted!

File: src/pages/buyer-signup/buyer-signup.html (hunk 4 above)
Comment (Codester291) [reply src/pages/buyer-signup/buyer-signup.html]: I'll resolve it

Comment (Codester291) [reply src/pages/buyer-signup/buyer-signup.html]: I'll resolve it

**Review by JuwonAbiola — APPROVED** [review]
good job

**Pull request merged**